presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -134,6 +134,9 @@
 
     private boolean projectionPushdownEnabled = true;
 
+    private boolean redirectingToIcebergEnabled;
```

Comment:
This would sound better written as "redirect" so the phrase is "redirect to Iceberg". I failed to come up with a precise reason why. Perhaps because "redirecting" is a gerund (verb functioning as a noun) rather than a noun.

This applies to all usages (config and session property).